src/tools/pkcs11-tool.c

```diff
@@ -1681,6 +1682,15 @@ static void show_token(CK_SLOT_ID slot)
 	printf("  serial num         : %s\n", p11_utf8_to_local(info.serialNumber,
 			sizeof(info.serialNumber)));
 	printf("  pin min/max        : %lu/%lu\n", info.ulMinPinLen, info.ulMaxPinLen);
+	printf("  uri                : pkcs11:model=");
+	printf("%s", percent_encode(info.model, sizeof(info.model)));
```

Comment:
this is really just a nit to simplify the maintenance in the future:
```suggestion
	printf("  uri                : pkcs11:");
	printf("model=%s", percent_encode(info.model, sizeof(info.model)));
```